src/ch00-11-counter.md

```diff
@@ -0,0 +1,14 @@
+# Simple Counter
+
+This is the Cairo adaptation of the [Solidity by example First Application](https://solidity-by-example.org/first-app/).
```

Comment:
I think we can skip the credits here, there's nothing very specific about a counter contract that would require it